### DISCUSSION TITLE
FHIR Pseudomizer als Alternative zum Deidentifhir

### DISCUSSION THREAD

**Discussion by trobanga** [medizininformatik-initiative/fts-next#429] [Ideas]
Der [FHIR pseudomizer](https://github.com/miracum/fhir-pseudonymizer) ist mit Sicherheit eine gute Alternative insb. da dieser ein externer Service ist und damit beliebig skalierbar ist. Da der Deidentifhir aber nicht unser Performance-Bottleneck ist, ist momentan der einzige nennenswerte Unterschied die Konfiguration zwischen Deidentifhir und Pseudomizer.
Es ist natürlich vorteilhaft, wenn die DIZen, die bereits den Pseudomizer verwenden, deren Konfiguration weiterverwenden können. Da könnten wir aber auch über einen Konfig-Converter nachdenken.

**Comment by SebStaeubert** [comment]
Dies ist eine Anforderung, die am 28.11.2024 in der MII Taskforce übergreifende Schnittstellen geäußert worden. Einige MII Standort nutzen aktuell den FHIR Pseudonymizer und könnten diesen auch verwenden, wenn sie FTSnext einsetzen.

**Comment by knoppiks** [comment]
Anbindung des Pseudomizers wurde erneut in der Sprechstunde nachgefragt.
